packages/runtime/src/widgets/Form/FormItem.tsx

```diff
@@ -20,35 +21,44 @@ export const EnsembleFormItem: React.FC<EnsembleFormItemProps<unknown>> = (
   const formInstance = AntForm.useFormInstance();
   const requiredRule = { required: Boolean(values?.required) };
 
+  const fieldLabel = useMemo(() => {
+    if (!values?.label) {
+      return null;
+    }
+
+    return (
+      <label
+        htmlFor={values.id ?? values.label}
+        style={{
+          ...values.labelStyle,
+        }}
+        title={values.label}
+      >
+        {isString(values.label)
+          ? values.label
+          : EnsembleRuntime.render([unwrapWidget(values.label)])}
+      </label>
+    );
+  }, []);
```

Comment:
no dependencies?

Reply:
No need to add dependencies here, just need to render one time on component render, and also when label will change the entire component will re-render and render correct value. Tested !!

Reply:
i see, values are coming from props